src/omnicore/notifications.h

```diff
@@ -3,26 +3,48 @@
 
 #include <stdint.h>
 #include <string>
+#include <vector>
+
+//! Alert types
+#define ALERT_FEATURE_UNSUPPORTED     0 // not allowed via alert broadcast
```

Comment:
Hmm, nope - I'm not really sure on best practices for programming :(  I just knew the compiler would replace those definitions with the values and thus it made the code easier to read - but if it's better practice to use ```const``` with the native type instead I can certainly start doing that :)